lib/build/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN clangParallelJobs="${clangParallelJobs:-$(nproc)}" && \
 FROM base as musl-builder
 WORKDIR $HOME/.symbolica/build
 COPY . .
-RUN apk add --no-cache git make && \
+RUN apk add --no-cache git make findutils && \
```

Comment:
Apparently the `xargs` that comes with busybox on Alpine is a bit basic and it seems to have a smaller limit on the size of the arg that can be passed to the command. So it looked like it was doing some batching of the `llvm-link` calls and so the output we were getting was just the linking off the objects in the final batch, which is why a bunch of stuff was missing.